packages/next/client/request-idle-callback.ts

```diff
@@ -30,4 +31,10 @@ const requestIdleCallback =
     }, 1)
   }
 
+export const cancelIdleCallback =
+  (typeof self !== 'undefined' && self.cancelIdleCallback) ||
+  function (id: RequestIdleCallbackHandle) {
+    return clearTimeout(id)
+  }
+
 export default requestIdleCallback
```

Comment:
Let's remove the old default export now?

Reply:
Didn't want to change more than I needed to, nor was I sure if this was a breaking change, but will do.

Reply:
This isn't a public API! It's for internal use only, break away!

Reply:
Done.